librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -1221,6 +1223,8 @@ crocksdb_options_set_max_sequential_skip_in_iterations(crocksdb_options_t*,
                                                        uint64_t);
 extern C_ROCKSDB_LIBRARY_API void crocksdb_options_set_disable_auto_compactions(
     crocksdb_options_t*, int);
+extern C_ROCKSDB_LIBRARY_API void crocksdb_options_set_disable_write_stall(
+    crocksdb_options_t*, bool);
```

Comment:
bool is not supported in C on MacOS. use unsigned char.

Reply:
I notice a lot of configs are using `bool`, e.g. `crocksdb_options_set_create_if_missing`

Reply:
`crocksdb_options_set_create_if_missing` is using `unsigned char`. https://github.com/tikv/rust-rocksdb/blob/master/librocksdb_sys/crocksdb/crocksdb/c.h#L985